### PR TITLE
docs: Added guide for Community Office Hours shepherding.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: bwplotka <bwplotka@gmail.com>

### CODE REVIEW

File: docs/contributing/community.md [hunk 1]
```diff
@@ -8,17 +8,39 @@ Follow and interact with us on Twitter: [@ThanosMetrics](https://twitter.com/Tha
...
-  * One week EU friendly time: 11:00-12:00 UTC
-  * One week US friendly time: 17:00-18:00 UTC
-* Calendar Event: https://www.cncf.io/calendar/
-* Agenda: [Google Doc](https://docs.google.com/document/d/137XnxfOT2p1NcNUq6NWZjwmtlSdA6Wyti86Pd6cyQhs)
-* Link to join zoom webinar: [https://zoom.us/j/96037860612](https://zoom.us/j/96037860612)
+**Everyone is welcome to join & add items to the agenda**.
 
 > NOTE: If no agenda items will be added 1h before the meeting, the meeting will be cancelled.
 
+**Meeting Time:** Every-second Thursday 15:00 CET
```
Comment (bwplotka) [inline docs/contributing/community.md:17]: Hour vote still pending (https://cloud-native.slack.com/archives/CL25937SP/p1659615345961049)

**Review by kakkoyun — APPROVED** [review]
:heart:

**Comment by Akshit42-hue** [comment]
Community Office Shepherds is really a good idea :)

File: docs/contributing/community.md (hunk 1 above)
Comment (matej-g) [reply docs/contributing/community.md:17]: Looks like 3PM still has the most votes, although there is some dissent in the thread

File: docs/contributing/community.md [hunk 2]
```diff
@@ -43,3 +65,52 @@ We participate in periodic mentorship programs. Read more [here](mentorship.md).
...
+| 2022.13 | TBD                      |
+| 2022.12 | TBD                      |
+| 2022.11 | TBD                      |
+| 2022.10 | TBD                      |
+| 2022.09 | `@bwplotka`              |
+| 2022.08 | `@bwplotka`              |
+
+Each month we rotate the role of Community Office Shepherds among Team Members. This role is focused on leading the safe and friendly discussions during our [Community Office Hours](#thanos-community-office-hours).
+
+> NOTE: Being shepherd does not mean other team members can chill out completely. (: It's a joined effort, so ideally there are multiple team members on those meetings to make decisions if needed.
```
Comment (matej-g) [inline docs/contributing/community.md:86]: nit:
```suggestion
> NOTE: Being shepherd does not mean other team members can chill out completely. (: It's a joint effort, so ideally there are multiple team members on those meetings to make decisions if needed. 
```

**Review by matej-g — DISMISSED** [review]
Beautiful :nail_care:

File: docs/contributing/community.md (hunk 1 above)
Comment (douglascamata) [reply docs/contributing/community.md:17]: What happens when the timezone changes from CET to CEST? We have to remember to come here and update this? 

What about writing time always in UTC to make it friendly for people from different timezones to understand at what time the meeting happens? It's easy to convert UTC to my "my own" timezone and it's much more difficult to convert "other random timezone" to "my own".

Even if it's decided that the meeting will happen in different times in European summer or winter, writing it down in UTC is still the most friendly way.

Comment (bwplotka) [reply docs/contributing/community.md:17]: Yea, I am always confused with UTC, plus it changes to maintainer team which is mostly CET (: - happy to change though.

**Pull request merged**